### DISCUSSION TITLE
Group based Index indicator

### DISCUSSION THREAD

**Discussion by ianrobertsFF** [pktharindu/nova-permissions#64] [Ideas]
I have rather a large number of permissions, as they need to be quite granular, it would be handy if there was an option for the indicator circles on the index component to have a grouped option, which would be green if all permissions were granted for that group, orange if only some, and red if none were granted. Potentially with a hover tooltip for more detail. 

I can probably pull together a PR at some point next week, just wondering if it was something you'd be open to?

**Comment by pktharindu** [comment]
Hey @ianrobertsFF, yeah, that sounds useful! Go for it!
